src/main/kotlin/ResponseBuilder.kt

```diff
@@ -0,0 +1,30 @@
+package response
+
+class ResponseBuilder(private val statusCode: String, private val body: String = "") {
```

Comment:
Why is `statusCode` a string instead of a number? Or, if Kotlin has enums, maybe it could be an enum.

Reply:
Changed to type `Int`. Using the proper type. Doing research into `enum`